libs/core/web/src/lib/context/AuthContext.tsx

```diff
@@ -0,0 +1,29 @@
+import {
+  createContext,
+  Dispatch,
+  ReactElement,
+  SetStateAction,
+  useState,
+} from 'react';
+
+export interface IAuthContext {
+  children: ReactElement;
+}
+
+interface ContextValue {
+  isLogged: boolean;
+  setIsLogged: Dispatch<SetStateAction<boolean>>;
```

Comment:
no need to expose this function outside of this component, you can just return isLogged